Cookbook/CookbookCommon/Sources/CookbookCommon/Recipes/WIP/PolyphonicSTK+MIDIKit.swift

```diff
@@ -0,0 +1,147 @@
+import AudioKit
+import AudioKitEX
+import AudioKitUI
+import AudioToolbox
+import Keyboard
+import SoundpipeAudioKit
+import STKAudioKit
+import SwiftUI
+import Tonic
+import MIDIKit
+import DunneAudioKit
+
+class PolyphonicSTKConductor: ObservableObject, HasAudioEngine {
+    let engine = AudioEngine()
+    let mixer = Mixer()
+    var notes = Array(repeating: 0, count: 11)
+    var osc = [RhodesPianoKey(), RhodesPianoKey(), RhodesPianoKey(), RhodesPianoKey(), RhodesPianoKey(),
+               RhodesPianoKey(), RhodesPianoKey(), RhodesPianoKey(), RhodesPianoKey(), RhodesPianoKey(), RhodesPianoKey()]
+    
+    // MIDI Manager (MIDI methods are in SoundFont+MIDI)
+    let midiManager = MIDIManager(
+        clientName: "TestAppMIDIManager",
+        model: "TestApp",
+        manufacturer: "MyCompany"
+    )
+    
+    var env: Array<AmplitudeEnvelope>
+    
+    var numPlaying = 0
+    func noteOn(pitch: Pitch, velocity: Int = 127) {
+        numPlaying += 1
+        if numPlaying > 10 {
+            numPlaying = 0
+        }
+        osc[numPlaying].trigger(note: MIDINoteNumber(pitch.intValue), velocity: MIDIVelocity(velocity))
+        notes[numPlaying] = pitch.intValue
+        env[numPlaying].openGate()
+    }
+    
+    func noteOn(pitch: Pitch, point _: CGPoint) {
+        noteOn(pitch: pitch, velocity: 120)
+        
+    }
+    
+    func noteOff(pitch: Pitch) {
+        for num in 0 ... 10 {
+            if notes[num] == pitch.intValue {
+                    env[num].closeGate()
+                    notes[num] = 0
+            }
+        }
+    }
+    
+    init() {
+        env = [AmplitudeEnvelope(osc[0]),AmplitudeEnvelope(osc[1]),AmplitudeEnvelope(osc[2]),AmplitudeEnvelope(osc[3]),AmplitudeEnvelope(osc[4]),AmplitudeEnvelope(osc[5]),AmplitudeEnvelope(osc[6]),AmplitudeEnvelope(osc[7]),AmplitudeEnvelope(osc[8]),AmplitudeEnvelope(osc[9]),AmplitudeEnvelope(osc[10])]
+
+        for envelope in env {
+            envelope.attackDuration = 0
+            envelope.releaseDuration = 0.2
+            mixer.addInput(envelope)
+        }
+
+        engine.output = mixer
+
+        // Set up MIDI
+        MIDIConnect()
+    }
+    
+    // Connect MIDI on init
+    func MIDIConnect() {
+        do {
+            print("Starting MIDI services.")
+            try midiManager.start()
+        } catch {
+            print("Error starting MIDI services:", error.localizedDescription)
+        }
+
+        do {
+            try midiManager.addInputConnection(
+                to: .allOutputs, // no need to specify if we're using .allEndpoints
```

Comment:
Super trivial detail, but the comment was probably made before a MIDIKit API change from `.allEndpoints` to `.allOutputs`. Could just update the comment to avoid any confusion.

Reply:
Thanks reviewing it! It's fixed now: https://github.com/AudioKit/Cookbook/commit/3b8f606d8dbff2680eac2c9afa382f81049b9f96